app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,139 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      Teaching computing is an important role to equip pupils with the knowledge and skills that they need to thrive in a world increasingly dependent on digital technology.  </p>
+
+<p>You will inspire pupils to access new fields and careers like quantum computing and artificial intelligence. You could inspire the next generation of computer scientists. </p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Help your pupils learn new skills",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0035.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0037.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>“It’s been really fantastic working with Year 6, watching them extend their coding skills, and seeing the delight when they do physical computing and see the results.” </p>
+
+<p><b>Stephen - Computing teacher at Stoke Poges School</b> </p>
+    <% end %>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>Computing for key stages 1 to 4 focuses on 3 pillars:</p>
+<ul>
+<li>information and communication technology  </li>
+
+<li>digital literacy </li>
+<li>computer science </li>
+</ul>
+<p>At GCSE and A Level you will teach computer science. You could teach the <a href="https://assets.publishing.service.gov.uk/media/5a7cb981ed915d682236228d/SECONDARY_national_curriculum_-_Computing.pdf">national curriculum</a>, with opportunities to develop your students’ computational thinking.  </p>
```

Comment:
See you guys said about confusing to say 'could' here but they actually do get to choose whether to follow the national curriculum or not so Rachael said 'Not all schools have to teach the national curriculum - its likely they’ll cover similar content but it’s not accurate to say they will definitely teach the NC.'